lib/promptConfig.js

```diff
@@ -29,7 +30,12 @@ const questions = (choicesList) => {
       type: 'input',
       name: 'description',
       message: 'Enter your commit message:',
-      validate: input => (input ? true : 'A commit message is mandatory!'),
+      validate: (input) => {
+        if (checkRulesEndWithDot(input) === false) return 'The commit message can not end with a dot';
+        else if (checkRulesMaxLength(input) === false) return `The commit message is not allowed to be longer as ${configuration.rules['max-char']}. Consider writing a body.`;
+        else if (checkRulesMinLength(input) === false) return `The commit message hast to be at least ${configuration.rules['min-char']} character.`;
+        return true;
```

Comment:
I think we should make this more flexible. If we add a rule we would probably not remember where to add some code. 

Maybe we can write it like following (I am open for better suggestions):

```js
const warnings = ruleWarningMessages(input);

if (warnings) return warnings;

return true;
```

With these functions we would never add any code again here, plus we can show more/all warnings at once. The current solution would always show one, although all of them would appear.

The rules could get configured like following:

```js
// lib/rules/availableRules.js
const rules = {
  endWithDot: 'The commit message...',
};
```

Other files:
`lib/rules/ruleWarningMessages.js`: in here we iterate over all available rules and check if the rule is set. The return would be a string with all messages in it (separated by `\n`). So if a new rule get added we never have to touch any function except `availableRules.js`.


What you think about it?